.azure/pipelines/jobs/default-build.yml

```diff
@@ -82,24 +79,20 @@ jobs:
     ${{ if ne(parameters.matrix, '') }}:
       maxParallel: 8
       matrix: ${{ parameters.matrix }}
-  # Map friendly OS names to the right queue
   # See https://github.com/dotnet/arcade/blob/master/Documentation/ChoosingAMachinePool.md
   pool:
-    ${{ if ne(parameters.poolVmImage, '') }}:
-      vmImage: ${{ parameters.poolVmImage }}
-    ${{ if and(eq(parameters.poolVmImage, ''), eq(parameters.agentOs, 'macOS')) }}:
+    # Map friendly OS names to the right queue
+    ${{ if eq(parameters.agentOs, 'macOS') }}:
       vmImage: macOS-10.13
-    ${{ if and(eq(parameters.poolVmImage, ''), eq(parameters.agentOs, 'Linux')) }}:
+    ${{ if eq(parameters.agentOs, 'Linux') }}:
       vmImage: ubuntu-16.04
-    ${{ if ne(parameters.poolName, '') }}:
-      name: ${{ parameters.poolName }}
-    ${{ if and(eq(parameters.poolName, ''), eq(parameters.agentOs, 'Windows')) }}:
-      ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+    ${{ if eq(parameters.agentOs, 'Windows') }}:
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
         name: NetCorePublic-Int-Pool
-        queue: BuildPool.Windows.10.Amd64.VS2019.BT.Open
```

Comment:
Are these pools new? What exactly do they / will they have on the agents?

Reply:
These pools already exist, but have VS 2019 Enterprise (and SQL Server).